admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -2,83 +2,165 @@
+Otherwise, take the approach that you're most comfortable with, likely the power
+user upgrade.
 
-7. If you keep your ``data/`` directory in your ``owncloud/`` directory, copy 
-   it from your old version of ownCloud to your new ``owncloud/``. If you keep 
-   it outside of ``owncloud/`` then you don't have to do anything with it, 
-   because its location is configured in your original ``config.php``, and 
-   none of the upgrade steps touch it.
+.. note::
+   Regardless of which approach that you take, they will both assume that your existing ownCloud installation is located in the default location: ``/var/www/owncloud``.
 
-8. If you are using 3rd party applications, look in your new ``owncloud/apps/`` 
-   directory to see if they are there. If not, copy them from your old ``apps/``
-   directory to your new one. Make sure the directory permissions of your third
-   party application directories are the same as for the other ones.
+The Standard Upgrade
+~~~~~~~~~~~~~~~~~~~~
 
-9. Restart your Web server.
+Delete all files and folders in your existing ownCloud directory (``/var/www/owncloud``) — **except** ``data`` and ``config``. 
 
-10. Now launch the upgrade from the command  line using ``occ``, like this 
-    example on CentOS Linux::
-    
-     sudo -u apache php occ upgrade
+.. attention:: Don't keep the ``apps`` directory.
+
+With those files deleted, extract the archive of the latest ownCloud server, over the top of your existing installation.
+
+::
+
+  # Extract the .tar.bz2 archive
+  tar -jxf owncloud-10.0.3.tar.bz2 -C /var/www/ --exclude config
+
+  # Extract the zip archive
+  unzip -q owncloud-10.0.3.zip -x "owncloud/config/*" -d /var/www/
+
+The Power User Upgrade
+~~~~~~~~~~~~~~~~~~~~~~
 
-  The optional parameter to skip migration tests during this step was removed in oC 9.2.
-  See :ref:`migration_test_label` for background information.
+Rename your current ownCloud directory, for example, from ``owncloud`` to ``owncloud-old``.
+Extract the unpacked ownCloud server directory and its contents to the location of your original ownCloud installation.
+::
 
-  See :doc:`../configuration/server/occ_command` to learn more.
+  # Assumes that the new release was unpacked into /tmp/
+  mv /tmp/owncloud /var/www/
+
+With the new source files now in place of the old ones, next copy the ``config.php`` file from your old ownCloud directory to your new ownCloud directory.
+::
+
+  cp /var/www/owncloud-old/config/config.php /var/www/owncloud/config/config.php
+
+If you keep your ``data/`` directory *inside* your ``owncloud/`` directory, copy it from your old version of ownCloud to your new version. 
+If you keep it *outside* of your ``owncloud/`` directory, then you don't have to do anything with it, because its location is configured in your original ``config.php``, and none of the upgrade steps touch it.
+
+If you are using 3rd party applications, look in your new ``/var/www/owncloud/apps/`` directory to see if they are there. 
```

Comment:
I wouldn't recommend to copy apps back before doing the upgrade below. IMHO its better to do this step after running the occ upgrade routine